analysis.ml

```diff
@@ -180,13 +180,16 @@ end
 
 exception DuplicateFormalParameter
 
-let as_var_set (formals : formal_parameter list) =
+let as_var_list (formals : formal_parameter list) =
```

Comment:
I know this is not really you code. But this looks a bit fishy. I think we should:
1. Check for duplicate parameters in the `check.ml` module and not here
2. only have one of the two `as_var_list` or `as_var_set` (I don't care which one) and then when you need something different the caller just needs to `VarSet.of_list` or `VarSet.elements`